docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -309,6 +397,116 @@ DESC StudentInfo;
 |                    age|      int|   NULL|
 +-----------------------+---------+-------+
 
+-- Drop columns of a table
+DESC StudentInfo;
++-----------------------+---------+-------+
+|               col_name|data_type|comment|
++-----------------------+---------+-------+
+|                   name|   string|   NULL|
+|                 rollno|      int|   NULL|
+|               LastName|   string|   NULL|
+|                    DOB|timestamp|   NULL|
+|                    age|      int|   NULL|
+|# Partition Information|         |       |
+|             # col_name|data_type|comment|
+|                    age|      int|   NULL|
++-----------------------+---------+-------+
+
+ALTER TABLE StudentInfo DROP columns (LastName, DOB);
```

Comment:
hmm, can users really run the example? AFAIK the builtin file sources do not support all the ALTER TABLE operations.

Reply:
@cloud-fan Thanks for the review. right, those statements I added except `RECOVER PARTITIONS` are not supported by builtin sources but users can technically use them by their custom file sources so I think they should be documented. I mentioned in a doc that they are only supported with v2 tables. Do you think they should not be in example?